src/validations/subscription.js

```diff
@@ -0,0 +1,23 @@
+const Joi = require('joi');
+
+const validateUpdate = (req, res, next) => {
+  const subscriptionValidate = Joi.object({
+    classId: Joi.string()
+      .hex()
+      .min(24),
+    members: Joi.string(),
+    date: Joi.date()
+      .max('now'),
```

Comment:
delete this validation. The user may submit suscriptions with a date after today